flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -131,7 +133,7 @@ public void testKeyedValueStateMigration() throws Exception {
                         new TestType.V2TestTypeSerializer()));
     }
 
-    @Test
+    @TestTemplate
     public void testKeyedValueStateSerializerReconfiguration() throws Exception {
```

Comment:
```suggestion
    void testKeyedValueStateSerializerReconfiguration() throws Exception {
```
Could be package private